odoo/addons/base/models/res_partner.py

```diff
@@ -172,7 +172,8 @@ class Partner(models.Model):
     _description = 'Contact'
     _inherit = ['format.address.mixin', 'avatar.mixin']
     _name = "res.partner"
-    _order = "display_name, id"
+    _order = "display_name ASC, id DESC"
+    _allow_sudo_commands = False
```

Comment:
[change] is 
_order = "display_name ASC, id DESC"
intended to be changed to?

Reply:
Existing in the commit I'm bringing. There are conflicts on those lines. Since the 175 line is committed after, I decided to bring it, but I could remove it as well...

Reply:
Yes, I had the same conflict. But with this, we change the behaviour the client had before and it is not from this change. Therefore I propose to not change it.

Reply:
okay! I will redo then :)